mobile/src/bundles/common/components/lost-connection-modal/lost-connection-modal.tsx

```diff
@@ -0,0 +1,49 @@
+import { useNetInfo } from '@react-native-community/netinfo';
+import React from 'react';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+
+import { Text, View } from '~/bundles/common/components/components';
+import { IconName, TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+const LostConnectionModal: React.FC = () => {
+    const { isConnected, isInternetReachable } = useNetInfo();
+    const isConnectionLos = !isConnected && !isInternetReachable;
+
+    if (!isConnectionLos) {
+        return null;
+    }
+
+    return (
+        <View
+            pointerEvents="box-only"
+            style={[
+                globalStyles.width100,
+                globalStyles.height100,
```

Comment:
why 100 here?

Reply:
It's an absolute element, which didn't let user change anything